layouts/partials/search-input.html

```diff
@@ -1,30 +1,82 @@
-{{ if or .Site.Params.gcs_engine_id .Site.Params.algolia_docsearch }}
-  <input
-    type="search"
-    class="form-control td-search-input"
-    placeholder="&#xf002; {{ T "ui_search_placeholder" }}"
-    aria-label="{{ T "ui_search_placeholder" }}"
-    autocomplete="off"
-  >
-{{ else if .Site.Params.offlineSearch }}
-  <div id="search-nav-container">
+<head>
+  <style>
+    .search-bar {
+      display: flex;
+      align-items: center;
+      background-color: #fff;
+      border: 1px solid #ccc;
+      border-radius: 20px; 
+      padding: 4px; 
+      width: 50vw;
+    }
+
+    .search-bar:focus-within {
+      border: 2.5px solid rgba(47, 135, 223, 0.7);
+    }
+
+    .search-bar i.fas {
+      margin-right: 10px; 
+    }
+
+    .td-search-input {
+      font-family: sans-serif;
+    }
+
+    .search-input {
+      flex: 1;
+      box-sizing: border-box;
+      border: none;
+      outline: none;
+      padding: 8px;
+      font-size: 16px;
+      border-radius: 15px;
+    }
+
+    .search-input::placeholder {
+      font-style: normal;
+      font-weight: normal;
+      font-size: 16px;
+      color: #aaa;
+    }
+  </style>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
```

Comment:
```suggestion
```
We already have this css integrated in the website

Reply:
done